### ISSUE TITLE
How to create a followup with the DialogFlow API

### ISSUE THREAD

**Issue by Gevert92** [googleapis/nodejs-dialogflow#479]
Please find a way to create a followup associated with an attempt using the DialogFlow API.

I think I'll try them with the code:
let intent = {
        displayName: dataIntent.nameIntent,
        trainingPhrases: trainingPhrases,
        messages: message
    };
    
let response = await IntentsClient.createIntent( {parent:parent, intent:intent} );

How can I create a followup?

**Comment by Gevert92** [comment]
I could fix it, I just had to add the fields rootFollowupIntentName and parentFollowupIntentName in the json:

let intent = {
        displayName: dataIntent.nameIntent,
        trainingPhrases: trainingPhrases,
        messages: message,
        rootFollowupIntentName: idIntentFollowup,
        parentFollowupIntentName: idIntentFollowup
 };

await IntentsClient.createIntent( {parent:parent, intent:intent} );

Where idIntentFollowup would be the identifier of the father intent example "projects/experiment-bot-257615/agent/intents/b87ad719-48e8-4938-9fa7-4fb7ff035d0e".

My confusion came because in the DialogFlow API documentation it said that the rootFollowupIntentName and parentFollowupIntentName fields are read-only, but I realized that this is not the case and we should not trust everything.

**Comment by bcoe** [comment]
:+1: awesome, it sounds like you're unblocked. Please feel free to open a new issue if you bump into any more hiccups.
